main.go

```diff
@@ -18,9 +18,10 @@ func main() {
 	err := cmd.Main(os.Stdout, os.Stderr, version, applySleepDurationString, os.Args)
 	if err != nil {
 		if errors.Is(err, commands.ErrBoshDiffChangesExist) {
-			log.Print(err)
+			fmt.Fprintln(os.Stderr, err)
```

Comment:
I'm coming from `om` maintainers past.

The context for the use of `log` was to ensure timestamps. Timestamps are always important in the automation flow.